extension/storage/filestorage/client.go

```diff
@@ -37,17 +37,17 @@ type fileStorageClient struct {
 	closed          bool
 }
 
-func bboltOptions(timeout time.Duration) *bbolt.Options {
+func bboltOptions(timeout time.Duration, noSync bool) *bbolt.Options {
 	return &bbolt.Options{
 		Timeout:        timeout,
-		NoSync:         true,
+		NoSync:         noSync,
```

Comment:
I'm happy to make that change.  My primary intention with exposing noSync as fSync was so that the default initialized config object would maintain the existing behavior, and this change wouldn't affect that goal.